### PR TITLE
#172888737 Create rooms

### PR DESCRIPTION
**Pull request by AbdoulNuru**
#### What does this PR do?
- implement create rooms in accommodation centres functionality
#### Description of Task to be completed?
- create a component to be used creating rooms
- test the functionality
#### How should this be manually tested?
- clone the repo, then checkout to this branch ``ft-create-rooms-172888737``
- run `npm i` to install all project dependencies
- run `npm run dev` to start the development server
- login as a travel administrator and on your dashboard go to add rooms to create a room in an accommodation facility, use credentials below to login
```
email: octopusbn@gmail.com
password: password
```
#### What are the relevant pivotal tracker stories?
[#172888737](https://www.pivotaltracker.com/story/show/172888737)
#### Screenshots (if appropriate)
Screen Shot 2020-05-20 at 01 06 29

### CODE REVIEW

**Review by blaise82 — APPROVED** [review]

**Review by hakoemmy — APPROVED** [review]
@AbdoulNuru Well done

**Review by Moise1 — APPROVED** [review]
Good job!

File: src/styles/signup.module.css [hunk 1]
```diff
@@ -12,9 +12,15 @@
...
 
 .forms_plus {
-  margin-left: 20px;
+  margin-left: 60px;
   margin-top: 50px;
-  width: 1500px;
+  width: 1300px;
+}
+
+.forms_plus2 {
```
Comment (AJAkimana) [inline src/styles/signup.module.css]: Use descriptive variable name

File: src/styles/signup.module.css [hunk 2]
```diff
@@ -33,6 +39,14 @@
   margin-left: auto;
 }
 
+.wrap3 {
```
Comment (AJAkimana) [inline src/styles/signup.module.css]: Descriptive variable name

File: src/styles/signup.module.css [hunk 3]
```diff
@@ -132,7 +146,7 @@
 
   opacity: 0.1;
 }
-.wrap1{
+.wrap1 {
```
Comment (AJAkimana) [inline src/styles/signup.module.css]: Use descriptive variable name

File: src/styles/signup.module.css [hunk 4]
```diff
@@ -202,7 +216,15 @@
     width: auto;
   }
 
-  .wrap,wrap1 {
+  .forms_plus2 {
```
Comment (AJAkimana) [inline src/styles/signup.module.css]: Use descriptive variable name

File: src/views/Dashboard/CreateRooms.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import Styles from "../../styles/entry.module.css";
+import styles2 from "../../styles/addAccom.module.css";
```
Comment (AJAkimana) [inline src/views/Dashboard/CreateRooms.js]: Use descriptive variable name

**Review by AJAkimana — CHANGES_REQUESTED** [review]
Name variables according to their function. So that in the future you will know what the variable, the function does without double-checking

**Review by AJAkimana — APPROVED** [review]

**Pull request merged**